tests/src/availability/concourse_test.go

```diff
@@ -0,0 +1,122 @@
+package availability_test
```

Comment:
I moved it to separate helpers package now and re-arranged which functions it contain. I've moved some from init_test.go as well.